app/src/main/java/com/asfoundation/wallet/service/Web3jKeystoreAccountService.java

```diff
@@ -37,21 +38,18 @@
   private static final int P = 1;
 
   private final KeyStoreFileManager keyStoreFileManager;
-  private final Scheduler scheduler;
   private final ObjectMapper objectMapper;
 
   public Web3jKeystoreAccountService(KeyStoreFileManager keyStoreFileManager, Scheduler scheduler,
```

Comment:
Since you remove the scheduler for all chains we can also remove in the constrcutor